examples/meta/src/evaluation/multiclassaccuracy.sg.in

```diff
@@ -0,0 +1,12 @@
+File ground_truth = csv_file("@SHOGUN_DATA@/label_train_multiclass.dat")
```

Comment:
Pls don’t use these files, but the ones that are labelled linear/nonlinear regression/classification